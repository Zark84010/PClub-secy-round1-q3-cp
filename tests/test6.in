1000 537 100
1922004 328224 889992 2127933 289300 463932 1435980 454464 291141 710363 217764 536520 676699 397656 643824 23670 356891 1251880 1409154 135708 583860 115720 849490 650925 298768 99414 1959876 1109334 1176399 2091902 41028 625940 58386 370830 213556 631200 1004134 268260 598325 590172 208296 1599829 1989595 54704 1446500 529419 1397319 136760 617787 999400 1792608 727195 35768 77322 440262 794523 42606 961002 1562220 236700 323490 151488 1855728 1639016 73640 1019388 1391796 1510146 154644 131500 549144 260370 21040 1288437 37872 539150 612264 229862 642772 324016 1872560 495755 563346 24722 598325 98625 1319208 1064624 486024 583860 451308 289826 159904 433950 1489632 299820 889992 144913 138864 228810 101255 740608 143598 75481 24985 678803 331380 459198 289300 1568532 2393300 499174 1076985 63120 1258455 1528030 735085 206192 73640 154644 175421 1157200 852909 248009 34716 3156 505486 124662 204351 319545 399234 40239 1287911 249850 414225 9731 1393900 1073040 1306584 622784 668546 849490 1026752 2156600 2551100 1167720 254058 2297568 475767 916555 2034305 870793 77848 689586 399760 300872 940751 157800 107041 995192 3682 268786 324016 526000 28930 126240 733244 45762 375564 1318945 138075 423430 2091902 693005 1785507 979149 588068 604900 463669 364518 2369367 598325 215660 194620 486024 124925 1472800 1498574 751128 689849 1788400 148332 393448 18410 896830 426060 35505 5523 491810 780321 686430 172791 1536183 87579 695898 35768 438158 1316315 840548 544410 12361 2293886 1196650 35768 1649536 1320786 594906 1352083 1778932 457620 593591 86790 80215 213556 657237 44973 2448530 1157200 31560 30508 21566 492336 820560 331380 978360 117824 39450 854750 678540 416592 20514 738504 1183500 1591676 2348064 520740 717990 894989 773220 234333 701158 288774 391344 215660 622258 1734748 717990 646191 381350 272205 93102 297453 12361 674595 236963 623310 1124851 896304 1107756 791104 427112 297453 220920 311129 273520 202510 1775250 618576 187782 2059816 687219 1587205 269838 112301 499700 94680 89946 1016232 1148258 739556 1988280 166216 228810 568080 15780 194883 608582 670650 1250828 1458598 1867300 189360 1178240 574655 934176 530208 1986439 2250228 244064 335588 1814700 656448 185152 409754 448415 676962 366622 13413 336640 127555 830817 109934 1176399 273520 1764204 48655 1701084 126240 686430 315600 375564 593591 29982 841600 142020 55230 1838896 220920 1633230 469718 130974 502067 146754 1854150 667494 1235574 867900 55230 1641120 1259244 468140 95469 449730 444996 919448 624888 564398 629096 54967 175158 942592 1412047 40765 1174032 769012 1751580 68380 80478 72588 158326 1009920 276150 482868 1312896 2091902 1058838 544936 354787 135708 841863 462880 1587205 537572 377405 1612716 807147 807936 415803 291930 362940 315600 785844 43395 636460 2265219 851857 994140 1391796 541254 157800 441577 520740 335588 833184 1458072 349790 2104 482868 126240 53389 541254 201195 911295 1863092 416592 1003608 1420200 20251 111775 105200 20777 142020 323490 923130 482868 302976 740608 199617 1639016 1073040 1316315 429216 213556 2048770 86264 187782 1789978 370304 663286 30245 1271868 850542 754810 29456 137286 2524800 1623762 1701084 1128270 231440 720357 387662 1026752 261948 1967766 701158 2010372 303765 93891 635934 305080 622258 593591 219868 289826 480238 162008 608582 298768 539676 54441 1126692 1494892 198828 1325520 683800 83897 1041480 257477 1928316 1842841 1393900 740608 2051400 1022018 250902 397656 1796553 2146080 121506 511009 419485 222498 2036146 1075144 907350 6838 2165016 95732 480238 865270 368200 347160 1311844 896830 1209537 187782 710889 1375227 473400 1070410 29456 888940 1531712 1766308 379772 457620 1533816 1319208 1005186 989669 208296 331380 1369704 177788 1841 697476 1538550 477345 99940 883680 133341 681170 1064361 721935 197250 260370 617787 883680 662760 306658 2249702 236700 622784 615420 1237152 287196 885258 736400 1384432 1034379 545462 549144 1117224 841074 992036 1483846 321912 14465 1299220 282725 315600 1065150 717990 1053052 441840 393448 74955 1639016 211452 232492 733770 54441 25248 157800 84949 1723176 1689512 1128796 430794 876316 484709 596484 532575 2002482 1108282 48918 473400 1588257 414225 426060 33138 234333 596484 970470 56808 594906 791104 60753 143598 338744 728247 393448 568080 937332 2524800 585175 1258455 607530 202510 603322 572288 1851520 1249250 1367600 683800 350579 366096 210400 1876242 110460 786896 532312 856328 226180 650925 263000 1009920 269838 239330 7890 894200 184100 151488 1249776 938910 69432 629622 260370 96258 15780 457620 17884 538098 36294 925760 472874 997296 12098 1920952 1367600 2059290 383454 1224265 68117 195672 307710 121506 1427301 1374175 239856 760070 61542 151488 173580 134919 846860 9669984 2316767 7252488 9669984 7051030 6043740 201458 7756133 5338637 7252488 4129889 2618954 4432076 7353217 9267068 9065610 2820412 6748843 7957591 1309477 5137179 9770713 2518225 3424786 1410206 8763423 10072900 7151759 9569255 6950301 1611664 4230618 4331347 4029160 3827702 6446656 9972171 9166339 4029160 302187 1410206 3223328 2921141 5842282 503645 7252488 4734263 8461236 3928431 2618954 9669984 4129889 2216038 7353217 2719683 3525515 5943011 2618954 5741553 5338637 4633534 7353217 4734263 4029160 1208748 8461236 402916 6245198 2618954 4331347 1712393 4029160 7554675 4532805 9770713 3122599 6849572 9569255 1410206 5842282 2014580 6144469 5137179 9367797 3324057 7756133 8058320 9166339 5439366 3626244 9669984 9972171 805832 4230618 3827702 7151759 7756133 4129889 8561965 201458 8461236 100729 9367797 1007290 9770713 4129889 4029160 6446656 3626244 503645 7453946 705103 6547385 2417496 5237908 5036450 5237908 3122599 4029160 503645 6648114 8763423 302187 2618954 8058320 9267068 4834992 805832 8561965 3223328 6245198 2014580 3223328 5439366 2921141 2820412 9569255 2014580 4432076 8259778 2417496 6950301 8864152 4029160 4432076 3928431 8964881 9569255 2115309 8159049 9972171 8763423 1913851 5338637 6446656 9972171 9669984 1108019 604374 8058320 4331347 6748843 9972171 2618954 7353217 7957591 5439366 6748843 9871442 4935721 4834992 7353217 1712393 8763423 1309477 6043740 7856862 100729 5439366 9871442 3324057 402916 3626244 5237908 805832 9972171 5036450 5540095 6245198 705103 3525515 402916 2618954 8561965 2921141 9871442 6446656 3424786 1611664 6144469 8259778 1510935 8662694 9871442 100729 9871442 906561 3021870 5036450 1410206 8058320 3525515 1712393 1510935 8662694 7655404 6648114 8763423 3122599 2719683 9367797 1712393 3021870 7051030 5338637 1007290 6748843 1611664 9669984 3424786 2921141 7756133 4834992 1410206 2719683 100729 6345927 8763423 3021870 6446656 100729 906561 9871442 1712393 7655404 8360507 4532805 4129889 2115309 2719683 1913851 6648114 9569255 10072900 3525515 4734263 906561 5439366 6345927 402916 8763423 4331347 3324057 3525515 805832 1108019 3525515 7051030 9770713 1611664 8561965 9770713 2518225 8360507 6648114 5237908 1712393 6245198 4432076 3827702 8864152 6245198 302187 8259778 6144469 8964881 8058320 2115309 4230618 9468526 2518225 2921141 3626244 906561 6345927 4331347 7151759 9770713 6446656 6748843 1208748 100729 1611664 8864152 3525515 3324057 9166339 5137179 9468526 3424786 4029160 8159049 9569255 9468526 1410206 5540095 8360507 4532805 7655404 2417496 3928431 5237908 5237908 7453946 1208748 6648114 6950301 8259778 1410206 8461236 10072900 7856862 3626244 
c 770 10341
f 108 663
f 392 711
c 588 6894
f 837 950
f 440 728
c 372 4596
f 419 923
f 535 986
c 698 383
f 68 846
f 336 504
c 133 11490
f 530 768
f 691 883
c 350 6128
f 423 697
f 969 988
c 767 9958
f 937 974
f 261 953
c 205 3064
f 984 999
f 722 984
c 128 14937
f 729 842
f 171 648
c 564 8426
f 661 844
f 617 948
c 293 12256
f 36 884
f 564 572
c 896 14937
f 661 749
f 377 666
c 888 5745
f 674 719
f 482 557
c 45 8809
f 281 453
f 210 404
c 367 5745
f 760 990
f 743 952
c 172 5362
f 959 990
f 423 763
c 91 8043
f 72 633
f 604 809
c 943 12639
f 13 141
f 808 930
c 895 14554
f 159 725
f 91 491
c 974 6894
f 866 924
f 422 897
c 401 12639
f 787 869
f 834 948
c 411 3447
f 632 634
f 129 686
c 459 2298
f 11 602
f 166 585
c 927 4979
f 239 370
f 706 811
c 833 11107
f 462 570
f 808 916
c 539 3447
f 140 726
f 471 956
c 543 13405
f 534 589
f 744 871
c 486 11107
f 721 871
f 141 435
c 984 8426
f 695 855
f 631 862
c 321 13405
f 539 585
f 493 763
c 686 11107
f 95 535
f 709 921
c 785 1149
f 88 468
f 158 729
c 549 6511
f 987 993
f 841 890
c 957 11490
