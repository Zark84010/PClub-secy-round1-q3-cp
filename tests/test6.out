141231
141231
54091473
537
537
537
537
537
537
537
537
54091473
54091473
537
54091473
537
537
537
537
537
537
537
537
537
537
141231
537
537
537
537
54091473
537
537
537
537
537
537
537
537
537
54091473
537
564924
537
537
537
537
537
537
537
537
537
537
537
537
537
537
537
537
537
537
537
537
537
54091473
537
